x-pack/packages/kbn-elastic-assistant/impl/assistant/prompt_textarea/index.tsx

```diff
@@ -26,35 +26,30 @@ const StyledTextArea = styled(EuiTextArea)`
 
 export const PromptTextArea = forwardRef<HTMLTextAreaElement, Props>(
   ({ isDisabled = false, value, onPromptSubmit, handlePromptChange, ...props }, ref) => {
-    const [currentValue, setCurrentValue] = React.useState(value);
```

Comment:
hoisted state to clear the message when the "send conversation" button is clicked